style: apply /style-guide pass to models/registry (#2721)
wandb/docs: models/registry/delete_registry.mdx

description: "Delete a W&B Registry as a Team admin or Registry admin using the Python SDK or the W&B App UI."
---

> This page shows how a Team admin or Registry admin can delete a registry, either programmatically with the Python SDK or interactively in the W&B App UI. Delete a registry when it's no longer needed so that it doesn't clutter your organization's Registry list.

Reviewer: ```suggestion
This page shows how a Team admin or Registry admin can delete a registry. An admins can programmatically or interactively delete a registry with the W&B Python SDK or the W&B App.
```

Also, which did we decide? "W&B App", "W&B App UI"...? These PRs are inconsistent

Author: Applied your suggestion (used "W&B App"; fixed "An admins" → "Admins"). On the naming question: the registry docs mix "W&B App" and "W&B App UI". I used "W&B App" where you flagged it, but a global decision is still needed — flagging for you and John.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

api = wandb.Api()

# Fetch the registry you want to delete

> fetched_registry = api.registry("[REGISTRY-NAME]")

Reviewer: See above

Author: Converted `[REGISTRY-NAME]` to `<registry-name>`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
